docs/RELEASE.md

```diff
@@ -1,3 +1,50 @@
+# Development Road Map
+The following is a summary of the NNI team's backlog for the next 6 months. Some completed items (either in a release or master branch) are included to provide the context and progress of the work.
+## New Tuners Supports
+* New MSRA research Algo
+* Hyperband
+* Grid search
+## New Training Services Supports
+* Kubeflow
+## User Interface and Experiences
+* Design nni APIs for users to specify search space in their code
```

Comment:
I wrote this sentence, but it is not accurate.
Support specifying neural architecture's search space through nni APIs easily.

Reply:
SDK supports for customize search space using NNI API?

Reply:
@QuanluZhang   @scarlett2018  Both of your updates look good for me,  I will copy from onenote after Mao's review.